code/datums/diseases/advance/symptoms/migraine

```diff
@@ -0,0 +1,36 @@
+/*
+//////////////////////////////////////
+
+Brain degradation
+
+	Gives brain damage over time and replaces the stat loss from the removal of the Aggressive Viral Metabolism
+
+//////////////////////////////////////
+*/
+
+/datum/symptom/migraine
+
+	name = "Brain degradation"
+	desc = "The virus slowly synthesizes Impedrezene, effectively removes the need of thought in one's actions."
```

Comment:
"...Impedrezene, **which** effectively removes..."

Reply:
fixed that